front_end/src/components/onboarding/steps/Step1.tsx

```diff
@@ -12,6 +13,18 @@ interface Step1Props {
 const Step1: React.FC<Step1Props> = ({ onTopicSelect, onClose }) => {
   const t = useTranslations();
 
+  useEffect(() => {
+    sendGAEvent({
+      event: "onboardingStarted",
+      category: "onboarding",
```

Comment:
It should actually be `event_category` to match our Custom definitions in GA

<img width="1075" alt="image" src="https://github.com/user-attachments/assets/8e94cb18-c6b7-4e58-a369-2dd8ecf0a9f4">

The same applies to other places where onboarding event is tracked